addons/website_sale_loyalty/static/tests/tours/test_website_sale_free_shipping_discount_line.js

```diff
@@ -1,11 +1,11 @@
 /** @odoo-module **/
 
 import { registry } from "@web/core/registry";
-import wsTourUtils from '@website_sale/js/tours/tour_utils';
+import wsTourUtils from "@website_sale/js/tours/tour_utils";
```

Comment:
On the contrary, we try to apply our [code guidelines](https://www.odoo.com/odoo/knowledge/8790) everywhere (in our scope). We usually use a double quote only for functional strings (which the user sees) and a single one for technical ones. Could you please revert them back in this file?
```suggestion
import wsTourUtils from '@website_sale/js/tours/tour_utils';
```

Reply:
Gotcha, somehow missed the quoting style also applied to JS.